tests/test_rebar.py

```diff
@@ -14,13 +14,13 @@
 from flask import Flask
 from werkzeug.routing import RequestRedirect
 
-from flask_rebar import HeaderApiKeyAuthenticator, SwaggerV3Generator
-from flask_rebar.authenticators import USE_DEFAULT
 from flask_rebar import messages
+from flask_rebar import HeaderApiKeyAuthenticator, SwaggerV3Generator
 from flask_rebar.compat import set_data_key
 from flask_rebar.rebar import Rebar
 from flask_rebar.rebar import prefix_url
 from flask_rebar.testing import validate_swagger
+from flask_rebar.utils.defaults import USE_DEFAULT
```

Comment:
Group imports and sort alphabetically

Reply:
`black` should take care of this, right?

Reply:
It doesn't no, there is actually a debate about it

Reply:
Well in that case I don't think we should care. I don't think we should add any additional formatting rules on top of `black`- IMO that would defeat the purpose.

Reply:
Fair enough. Maybe [isort](https://github.com/timothycrosley/isort) for offering a standard opinion on these?

Reply:
Yeah I looked at isort, it seems nice but I remember seeing some incompatibilities with black (maybe resolved by now).